generator/vulkan/render.zig

```diff
@@ -728,9 +728,20 @@ fn Renderer(comptime WriterType: type) type {
 
                 try self.writer.writeAll(" = .");
                 try self.writeIdentifierWithCase(.snake, stype["VK_STRUCTURE_TYPE_".len..]);
+            } else if (field.field_type == .name and !container.is_union and mem.eql(u8, "VkBool32", field.field_type.name) and isFeatureStruct(container.extends)) {
```

Comment:
These are honestly more checks than required, I think - but I'm trying to narrow it down a bit with less expensive checks before doing the expensive `mem.eql`s.